scripts/gulp/spacings.js

```diff
@@ -29,20 +31,24 @@ const VALID_SPACINGS = new Set([
   'icontext',
 ]);
 
-const spacings = (properties, formatName, getLegibleName) => _.chain(properties)
+const filterValidAndMap = (properties, formatName) => _.chain(properties)
```

Comment:
The function name has changed here could we either add some comments above for documentation on what this function does or change the name so its understandable what this does as the new name `filterValidAndMap` is not very descriptive  reading this 'valid', what is valid code, values, numbers?